app/lib/actions/katello/organization/simple_content_access/prepare_content_overrides.rb

```diff
@@ -0,0 +1,36 @@
+module Actions
+  module Katello
+    module Organization
+      module SimpleContentAccess
+        class PrepareContentOverrides < Actions::Base
+          def plan(organization_id)
+            Rails.logger.info "PrepareContentOverrides plan: #{organization_id.inspect}"
+            organization = ::Organization.find(organization_id.to_i)
+            org_name = organization.name
+
+            plan_self(organization_id: organization_id, organization_name: org_name)
+          end
+
+          def run
+            organization = ::Organization.find(input[:organization_id].to_i)
+            migrator = ::Katello::Util::ContentOverridesMigrator.new(organization: organization)
+
+            output[:migrator_result] = migrator.execute_non_sca_overrides!
+          end
+
+          def rescue_strategy
+            Dynflow::Action::Rescue::Skip
+          end
```

Comment:
@wbclark : Hoping to piggyback off your testing here. Once you're done with the happy path testing, can you force an error in app/lib/katello/util/content_overrides_migrator.rb > execute_non_sca_overrides! method and test if Dynflow::Action::Rescue::Skip makes sense as rescue strategy. I think org SCA will end up being enabled even if there's an error in the overrides method cause the output of first action isn't passed to second action so skip chaining won't work..

@jeremylenz : Should we just remove this and not skip and fail instead?

Reply:
If it can't create the overrides for some reason, I _want_ SCA to still be enabled. That should be what Skip does, right?

Reply:
>> That should be what Skip does, right?

Yes..So SCA will be enabled without the overrides? Is there a way to go back and create those overrides?

Reply:
Well... you could've back when this was a rake task :grinning: 

I don't anticipate that this will fail except in very rare circumstances, so I don't really consider that a problem.

Reply:
Also, you can always create overrides manually if you want (the "normal" way)

Reply:
Oh actually.. You can just disable and enable SCA again!